Client/src/Pages/Status/index.jsx

```
@@ -1,35 +1,85 @@
 const Status = () => {
 	const theme = useTheme();
+	const navigate = useNavigate();	
+	const {authToken} = useSelector((state) => state.auth);
+	const [initForm, setInitForm] = useState({});
+	const STATUS_PAGE = import.meta.env.VITE_STATU_PAGE_URL?? "status-page";					
+	useEffect(() => {
+		const getStatusPage = async () => {
+			let config = { authToken: authToken, url: STATUS_PAGE };
+			try {
+				let res = await networkService.getStatusPageByUrl(config);			
+				if(res && res.data)
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)